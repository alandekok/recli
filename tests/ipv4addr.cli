ipv4 1.2.3.4
ipv4 0.1.2.3
ipv4 -3.3
ipv4 xyz
ipv4 333.0.1.2
ipv4 1.2.3.3a
