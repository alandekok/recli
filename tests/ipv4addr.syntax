ipv4 IPV4ADDR
