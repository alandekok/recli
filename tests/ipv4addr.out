ipv4 -3.3
     ^ Input does not match required syntax.
ipv4 xyz
     ^ Input does not match required syntax.
ipv4 333.0.1.2
     ^ Input does not match required syntax.
ipv4 1.2.3.3a
     ^ Input does not match required syntax.
